build/teamcity-acceptance.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -exo pipefail
```

Comment:
done (I initially wanted to copy it verbatim from the TC config, changing only where the behavior was defined rather than the behavior itself)